oaknational/Oak-Web-Application
# [PUPIL-805] Refactoring some areas of the pupil journey

## Description

Music year: 1966

This branch refactors a few areas of the pupil browse journey.

I'll add comments throughout the PR for code review.

## How to test

1. Go to https://deploy-preview-2541--oak-web-application.netlify.thenational.academy/pupils/years
2. Check that the browse journey is working as expected

## Checklist
